### DISCUSSION TITLE
https://notes.widcard.win/postgraduate/cmu/db-15-445/c02-intermediate-sql

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#18] [Announcements]
# https://notes.widcard.win/postgraduate/cmu/db-15-445/c02-intermediate-sql

Notes of widcardw

https://notes.widcard.win/postgraduate/cmu/db-15-445/c02-intermediate-sql

**Comment by AzelaicAcid** [comment]
鼓励！
